stages/org.osbuild.users

```diff
@@ -94,6 +94,7 @@ def useradd(root, name, uid=None, gid=None, groups=None, description=None, home=
         arguments += ["--comment", description]
     if home:
         arguments += ["--home-dir", home]
+        arguments += ["--create-home"]
```

Comment:
The `useradd` man page (shadow-utils v4.8.1) on Fedora 34 for the `-d, --home-dir HOME_DIR` option says:
The new user will be created using HOME_DIR as the value for the user's login directory. The default is to append the LOGIN name to BASE_DIR and use that as the login directory name. **If the directory HOME_DIR does not exist, then it will be created unless the -M option is specified.**

So I would expect that if `home` is specified, then the `--home-dir` alone should do the trick. Does Arch uses some different version of shadow-utils?

Reply:
I think it depends on `CREATE_HOME` in `/etc/login.defs` (and maybe some built-in defaults). RHEL and Fedora do have that option so specifying it won't hurt. Alternatively we could have a global stage option (a sibling to `users`) called `create-home` (or something) that we could enable for Arch.

Reply:
Yes, Arch does not set `CREATE_HOME` which means a `useradd --home-dir /home/testing` does not create a home directory.

Reply:
Thanks for the additional details. I agree that adding the option won't hurt 😉